src/v3/src/components/AuthenticatorButton/AuthenticatorButton.tsx

```diff
@@ -97,14 +93,19 @@ const AuthenticatorButton: UISchemaElementComponent<{
       component="button"
       type={type}
       sx={(theme) => ({
-        '--ColorHover': theme.palette.primary.dark,
-        '--ColorPrimaryBase': theme.palette.primary.main,
```

Comment:
is this style being set by the odyssey theme?

Reply:
we were previously using CSS variables (anything with the `--` prefix) by setting the value of that variable here and then using them in the `AuthenticatorButton.module.css` file (now deleted). I just changed it so we directly use the theme color value in `sx` and eliminated the class we were adding to do the styling.